WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
I'm not removing all conditions. There are still significant conditions for making a dynamic realm that require significant work. Even the player sweats somewhat to get to this. Now, imagine the AI trying it.